---
title: "Species-specific behavior and ecology drive depth distribution of rare species eDNA in pelagic marine environments"
author: "Amy M. Van Cise^1\\*^, Eiren K. Jacobson^2^, Megan Shaffer^3^, Pedro Brandao^3^, David Miller^4^, Ryan Kelly^3^, Brice Semmens^5^,  Kim M. Parsons^6^"
format: html
editor: visual
execute:
  echo: false
  message: false
  warning: false
project:
  output-dir: Docs
bibliography: references.bib
---

1.  School of Aquatic and Fishery Sciences, University of Washington, Seattle, WA, USA
2.  Centre for Research into Ecological and Environmental Modelling, University of St Andrews, St Andrews, Scotland
3.  School of Environmental and Marine Affairs, University of Washington, Seattle, WA, USA
4.  Dave's affiliation
5.  Scripps Institution of Oceanography, University of California San Diego, San Diego, CA, USA
6.  Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, Seattle, WA, USA

<br>

^\*^Corresponding author email: avancise\@gmail.com

<br>

Running page head: Depth distribution of rare target detections

```{r}
library(tidyverse)
library(patchwork)
library(cowplot)

# load data
load("ProcessedData/detect_data.RData")
load("ProcessedData/H1models.RData")

# load plots
load("Figures/sampling_map.Rdata")
load("Figures/detectDepth_plots.Rdata")
load("./Figures/H1PODplots.Rdata")
load("./Figures/H3.0c_map.Rdata")

metadata <- read.csv("./Data/Hake_2019_metadata.csv")
```

## Introduction

All analyses of community composition face a similar challenge: rare species dropout. This leads to inaccurate estimates of biodiversity or community composition and greater difficulty assessing changes in ecosystem resilience, as well as inaccurate characterization of a rare species distribution and greater difficulty assessing changes in distribution in response to environmental perturbations.

Many methodological approaches have been developed to address this problem, both in the experimental design phase (REFS) and in the data analysis phase (REFS). Statistical approaches aim to quantify and minimize the effect of rare species dropout, while sampling approaches aim to reduce the rate of dropout via robust sampling design. Within the latter category, it has been shown that adopting an ensemble approach, i.e. integrating across multiple types of observation methods in order to smooth across the weaknesses of each approach individually, often produces more robust estimates of species' distributions and community composition than using single methods alone (REFS).

eDNA is emerging as a viable tool for improving spatial distribution models for rare marine species, especially when used jointly with existing sampling methods, e.g. net tows, active acoustics, and/or bottom trawls for fishes and invertebrates. In the case of marine mammals - notably some of the rarest species in the marine environment - two survey techniques are traditionally used. Visual line-transect surveys are the most common, in which observers aboard a ship or airplane will record detections of any marine mammals and their distance to the transect line, covering the survey area in a systematic pattern. These surveys are limited to the surface of the ocean, where most marine mammals spend very little of their time, and to daylight hours - and their effectiveness decreases in poor weather or sea state. Ship-board passive acoustic line-transect surveys collect recordings along similar systematic transects, and can often detect species up to XX kilometers away from the ship in any direction. They have some of the same limitations at visual surveys, but only detect species that are actively vocalizing. Systematic eDNA surveys, where water samples are collected along transects, do not share limitations related to depth, daylight, weather conditions, or species' vocal activity, but notably have a detection range that is much smaller than either of the two previous surveys and strongly affected by diffusion and transport by oceanographic features (REFS).

In open ocean, deep water environments, our understanding of species' distributions may be affected by sampling depth, a fundamental aspect of experimental design often overlooked in 3D environments. This is especially true for mobile species capable of utilizing a broad swath of the water column. In the pelagic marine environment, some rare species have depth distributions spanning 3km or greater, which complicates decision-making regarding where to collect point-samples with the aim of describing x-y distribution, either independently of or alongside z-distribution.

Because eDNA samples have a limited radius of detection, and because they are costly both to collect and to process, studies targeting a specific set of rare species may benefit from restricting their range of sampling depths, especially when describing x-y distribution has a higher priority than describing z-distribution. However, there is currently a lack of published data providing guidance regarding the range of sampling depths that can be used to target certain species, taxonomic groups, or ecological groups of rare species, or what ecological or observational processes may drive variability in the probability of detecting specific species across depths.

Here, we aim improve our understanding of the depth distribution of rare species' eDNA in the water column and identify major environmental and ecological drivers of observed patterns. We specifically ask three questions:

1.  Does cetacean eDNA distribution vary in the z-dimension? If so, what major ecological factors (e.g. preferred prey, time spent at depth, or phylogeny) can be used to predict cetacean eDNA distribution across depths?
2.  Do common sources of observational bias, e.g. technical replication, choice of primers, or sampling volume, interact with sampling depth to affect probability of detection?
3.  Does x-y variability in eDNA distribution interact with depth variability to affect probability of detection?

To answer these questions, we utilize eDNA samples collected in the California Current from the northern extent of the United States' EEZ to the latitude of XXX in southern California, and between XXX and XXX miles offshore. By collapsing eDNA detections across x and y dimensions and focusing only on the z-distribution of sampling effort (0-500 m), we aim to determine (1) whether species- or taxon-specific detection probability varies with depth and (2) how depth-dependent detection probability varies with increasing sample volume, primer specificity, and technical replication. Finally, we explicitly re-introduce x-y dimensions in order to determine (3) whether depth-dependent detection probability varies with latitude or longitude.

## Methods

{{< include zDist_methods.qmd >}}

## Results

{{< include zDist_results.qmd >}}

## Discussion

#### Question 1:

1.  Sampling depth matters. To target rare species, optimizing sampling depth could increase detection rates and improve xy distributional models while minimizing cost.

2.  Sampling depth varies by species, and roughly according to some basic ecological principles, e.g. generally eDNA was found where the animals spend most of their time, and prey type was indicative of POD across depth.

3.  Notable caveats are: whalefall and poop (the latter mostly noticeable for baleen whales). *Include some figures of potential whalefall here or in supplement? Or add this as an explicit method/result? Include poop sinking equations here or in a supplement?*

4.  Beaked whale distributions are poorly understood and could be improved the most by eDNA sampling/monitoring. Talk here about how eDNA depth distributions compare to what we know about beaked whale distributions, e.g. two of the detected species are found most often around 300m, where they are thought to take resting dives. (Is this where they spend most of their time according to Navy reports?) Stejnegers was found deeper. Sampling deeper than 500m may increase detection for deeper diving beaked whales, e.g. some of the ones that are almost never detected at the surface. *EKJ note: do we think this is because beaked whales are rare rare targets, so their eDNA has a better chance of being detected at depth, where there are fewer competing bits of other cetacean eDNA?)*

#### Question 2:

5.  Increasing technical replication (and primer specificity?) increases POD. *and this covaries with depth effect?*

#### Question 3:

6.  Effect of depth varies across x and y dimensions. EKJ note: if we find this, it would be indicative of oceanographic stuff (upwelling etc) affecting probability of detection

7.  Over the large geographic range covered by this study and the large range of mobile rare species, we expect that this at least partially reflects true distribution of the animals.

8.  It could also reflect, e.g. shorter residence time of eDNA in warmer water to the south, or higher probability of eDNA sinking below 500 meters in deeper offshore water. *What else could be causing this?*

ASSUMPTIONS AND CAVEATS
