---
format: html
editor: visual
---

### Sample Collection


For this study, we use previously collected water samples acquired during biennial NOAA surveys quantifying hake abundance and distribution \[\@shelton_etal22\]. Samples were collected in tandem with nightly CTD casts, as described in Shelton et al. \[-\@shelton_etal22\]. We targeted 0, 50, 150, 300, and 500 m depth strata along east-west transects with a western terminus at XXX and an eastern terminus at the continental shelf break, generally \~XX km from shore (Figure 1). 

The study area is encompassed by the California Current System, which is 500-800km wide, extends from 43°N to 23°N, and comprises an equatorward surface current and a subsurface poleward current that is concentrated mainly on the continental shelf [@hickey79]. The system is characterized by strong alongshore advection, inshore upwelling concentrated primarily along the continental shelf break, and multiple terrestrial and oceanic water sources [e.g. @mcclatchie14, @auad_etal11]. These complex oceanogrpahic processes support a highly productive and diverse assemblage of plankton and fishes, form a rich prey base for at least 25 cetacean species that inhabit the study area [@carretta_etal23].

### Sample Processing

We extracted DNA from all samples using a phenol-chloroform extraction protocol as described in XXX (Shaffer? Valdivia? another?). We then amplified each sample using three different primers: MiFish Universal (hereafter MFU, REF), targeting the 12S region of fishes, which often captures marine mammals; Marine Vertebrates 1 (hereafter MarVer1, REF), targeting the 12S region of marine vertebrates broadly; and D-loop (hereafter DL, REF) targeting the control region of cetaceans specifically. We amplified one technical replicate for MFU, and three technical replicates each for DL and MV1. Our PCR amplification protocols ... **MORE HERE FROM MEGAN?**

### Bioinformatic Processing

All raw sequence data were processed using a custom dada2-based pipeline and custom reference database based on NCBI sequence data. The pipeline and code for reference database generation can be found at \[GITHUB REPO HERE\]. Briefly, we ... **MORE HERE FROM PEDRO?**

Once the raw sequences were trimmed, filtered, merged, clustered to ASVs, and once chimeras were removed, we assigned ASV taxonomy using ... **MORE HERE FROM PEDRO?**

For each primer region, we built a custom reference database using all available sequence data from NCBI with the following filtering parameters. First, because the 12S mtDNA region has generally low resolution across many cetacean species, we filtered the reference database to exclude any species not observed in the North Pacific Ocean, excepting a few species known to occasionally visit from nearby regions, e.g. bowhead and beluga whales. We also exclude any of NCBI's sequence IDs that were not manually validated by a human, due to the fact that NCBI's automated ID pipeline is still in beta mode and can produce erroneous identifications (REF here?).

### Data QAQC

We defined a sample as a station/depth/technical replicate combination, and first removed any samples considered to be sequencing failures. Sequencing failures were defined for MV1 and MFU as samples for which no fish were successfully sequenced. Because DL targets cetaceans and excludes fish, we did not define a sequencing failure for this primer and instead included all samples run. Positive detections were defined as samples containing at least one cetacean DNA sequence for each of the three primers, following XXX (REF HERE). We filtered out any non-target (non-cetacean) species, and also removed any detections of bottlenose dolphins and unidentified Balaenopterids or Delphinids (because ... ).

To obtain non-detections, we included all MFU and MV1 samples that had at least one marine species detected, but no cetaceans - excluding any samples that completely failed sequencing. Because our DL primer specifically targets a subset of very rare species, we assume that sequencing failure indicates a non-detection - therefore, for this primer we included any station/depth/species combination that was not detected as a non-detect.

### Data Analysis

#### Question 1: Does POD vary with depth?

We addressed this question using generalized additive models (GAMs; @Wood2017) implemented using the mgcv package in R [@RCoreTeam2016]. GAMs have previously been demonstrated to be an appropriate tool for modeling spatial use in cetaceans (e.g. @Becker2016; @becker2020; @becker2022; @Becker2019; @abrahmsetal_19c). We first collapsed our detection data across all technical replicates at each sampling station and depth and defined a positive detection at that location and depth as detection in at least one technical replicate. We fit all GAMs using 0/1 detections as the response variable with a binomial distribution and logit link function, and estimated model parameters using REML. Models examining depth effect alone were built iteratively, starting with (1) assessing probability of detection (POD) across all species at sampling depths between 1 and 500m, then (2) allowing y-intercept to vary by species, and finally (3) allowing model smooth to vary by species.

In addition to examining depth variability across species-level strata, we examined several other ecological stratification schemes in order to determine whether depth variability in detection probability is driven by (1) evolutionary inertia (i.e. taxonomic family) or (2) preferred prey type.

We assessed model fit using AIC, and model performance using standard metrics including the percent of deviance explained, the area under the receiver operating characteristic curve (AUC; Fawcett 2006 REF), the true skill statistic (TSS, Allouche et al. 2006 REF), and a visual comparison of predicted vs. observed detection rates across sampling depths. We also visually compared our model-based predictions of probability of detection across depth with estimates of species-specific time-at-depth as reported in [@watwood_buonantony12] and [@u_s_departmentofthenavy17].

#### Question 2: Does depth variability in POD covary with observation bias?

To address this question, we extracted smooths from the best fit model in Q1 and input them into a Bayesian hierarchical occupancy model, developed based on Patin et al. (*in prep*) to explicitly examine depth variability alongside potential sources of observational bias, including technical replication, primer specificity, sampling volume, and number of freeze/thaw cycles. **MORE HERE**

#### Question 3: Does depth variability in POD covary with xy variability?

We returned to the use of GAMs to address Question 3, and again built iterative models in order to examine covariability between depth, latitude, and longitude on a per-species basis. We began by examining whether xy variability explained a significant amount of remaining variability left in the model after depth variability was account for (i.e. POD \~ z + xy), then tested whether x, y, and z covary (i.e. POD \~ xyz). Finally, we allowed the model of xyz covariability to vary idependently by species.

We assessed model fit using AIC, and model performance using percent deviance explained, AUC, and TSS, as in Question 1. We then conducted a visual comparison of the predicted depth of maximum POD across and our observed detection depth within the study area for the most frequently observed baleen whale, dolphin, and beaked whale.