<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Amy M. Van Cise1*, Eiren K. Jacobson2, Megan Shaffer3, Pedro Brandao3, David Miller4, Ryan Kelly3, Brice Semmens5, Kim M. Parsons6">

<title>Species-specific behavior and ecology drive depth distribution of rare species eDNA in pelagic marine environments</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-8a79a254b8e706d3c925cde0a310d4f0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Species-specific behavior and ecology drive depth distribution of rare species eDNA in pelagic marine environments</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Amy M. Van Cise<sup>1*</sup>, Eiren K. Jacobson<sup>2</sup>, Megan Shaffer<sup>3</sup>, Pedro Brandao<sup>3</sup>, David Miller<sup>4</sup>, Ryan Kelly<sup>3</sup>, Brice Semmens<sup>5</sup>, Kim M. Parsons<sup>6</sup> </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<ol type="1">
<li>School of Aquatic and Fishery Sciences, University of Washington, Seattle, WA, USA</li>
<li>Centre for Research into Ecological and Environmental Modelling, University of St Andrews, St Andrews, Scotland</li>
<li>School of Environmental and Marine Affairs, University of Washington, Seattle, WA, USA</li>
<li>Dave’s affiliation</li>
<li>Scripps Institution of Oceanography, University of California San Diego, San Diego, CA, USA</li>
<li>Northwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, Seattle, WA, USA</li>
</ol>
<p><br></p>
<p><sup>*</sup>Corresponding author email: avancise@gmail.com</p>
<p><br></p>
<p>Running page head: Depth distribution of rare target detections</p>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>All analyses of community composition face a similar challenge: rare species dropout. This leads to inaccurate estimates of biodiversity or community composition and greater difficulty assessing changes in ecosystem resilience, as well as inaccurate characterization of a rare species distribution and greater difficulty assessing changes in distribution in response to environmental perturbations.</p>
<p>Many methodological approaches have been developed to address this problem, both in the experimental design phase (REFS) and in the data analysis phase (REFS). Statistical approaches aim to quantify and minimize the effect of rare species dropout, while sampling approaches aim to reduce the rate of dropout via robust sampling design. Within the latter category, it has been shown that adopting an ensemble approach, i.e.&nbsp;integrating across multiple types of observation methods in order to smooth across the weaknesses of each approach individually, often produces more robust estimates of species’ distributions and community composition than using single methods alone (REFS).</p>
<p>eDNA is emerging as a viable tool for improving spatial distribution models for rare marine species, especially when used jointly with existing sampling methods, e.g.&nbsp;net tows, active acoustics, and/or bottom trawls for fishes and invertebrates. In the case of marine mammals - notably some of the rarest species in the marine environment - two survey techniques are traditionally used. Visual line-transect surveys are the most common, in which observers aboard a ship or airplane will record detections of any marine mammals and their distance to the transect line, covering the survey area in a systematic pattern. These surveys are limited to the surface of the ocean, where most marine mammals spend very little of their time, and to daylight hours - and their effectiveness decreases in poor weather or sea state. Ship-board passive acoustic line-transect surveys collect recordings along similar systematic transects, and can often detect species up to XX kilometers away from the ship in any direction. They have some of the same limitations at visual surveys, but only detect species that are actively vocalizing. Systematic eDNA surveys, where water samples are collected along transects, do not share limitations related to depth, daylight, weather conditions, or species’ vocal activity, but notably have a detection range that is much smaller than either of the two previous surveys and strongly affected by diffusion and transport by oceanographic features (REFS).</p>
<p>In open ocean, deep water environments, our understanding of species’ distributions may be affected by sampling depth, a fundamental aspect of experimental design often overlooked in 3D environments. This is especially true for mobile species capable of utilizing a broad swath of the water column. In the pelagic marine environment, some rare species have depth distributions spanning 3km or greater, which complicates decision-making regarding where to collect point-samples with the aim of describing x-y distribution, either independently of or alongside z-distribution.</p>
<p>Because eDNA samples have a limited radius of detection, and because they are costly both to collect and to process, studies targeting a specific set of rare species may benefit from restricting their range of sampling depths, especially when describing x-y distribution has a higher priority than describing z-distribution. However, there is currently a lack of published data providing guidance regarding the range of sampling depths that can be used to target certain species, taxonomic groups, or ecological groups of rare species, or what ecological or observational processes may drive variability in the probability of detecting specific species across depths.</p>
<p>Here, we aim improve our understanding of the depth distribution of rare species’ eDNA in the water column and identify major environmental and ecological drivers of observed patterns. We specifically ask three questions:</p>
<ol type="1">
<li>Does cetacean eDNA distribution vary in the z-dimension? If so, what major ecological factors (e.g.&nbsp;preferred prey, time spent at depth, or phylogeny) can be used to predict cetacean eDNA distribution across depths?</li>
<li>Do common sources of observational bias, e.g.&nbsp;technical replication, choice of primers, or sampling volume, interact with sampling depth to affect probability of detection?</li>
<li>Does x-y variability in eDNA distribution interact with depth variability to affect probability of detection?</li>
</ol>
<p>To answer these questions, we utilize eDNA samples collected in the California Current from the northern extent of the United States’ EEZ to the latitude of XXX in southern California, and between XXX and XXX miles offshore. By collapsing eDNA detections across x and y dimensions and focusing only on the z-distribution of sampling effort (0-500 m), we aim to determine (1) whether species- or taxon-specific detection probability varies with depth and (2) how depth-dependent detection probability varies with increasing sample volume, primer specificity, and technical replication. Finally, we explicitly re-introduce x-y dimensions in order to determine (3) whether depth-dependent detection probability varies with latitude or longitude.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<section id="sample-collection" class="level3">
<h3 class="anchored" data-anchor-id="sample-collection">Sample Collection</h3>
<p>For this study, we use previously collected water samples acquired during biennial NOAA surveys quantifying hake abundance and distribution [@shelton_etal22]. Samples were collected in tandem with nightly CTD casts, as described in Shelton et al.&nbsp;[-@shelton_etal22]. We targeted 0, 50, 150, 300, and 500 m depth strata along east-west transects with a western terminus at XXX and an eastern terminus at the continental shelf break, generally ~XX km from shore (Figure 1).</p>
<p>The study area is encompassed by the California Current System, which is 500-800km wide, extends from 43°N to 23°N, and comprises an equatorward surface current and a subsurface poleward current that is concentrated mainly on the continental shelf <span class="citation" data-cites="hickey79">(<a href="#ref-hickey79" role="doc-biblioref">Hickey 1979</a>)</span>. The system is characterized by strong alongshore advection, inshore upwelling concentrated primarily along the continental shelf break, and multiple terrestrial and oceanic water sources <span class="citation" data-cites="mcclatchie14">Auad, Roemmich, and Gilson (<a href="#ref-auad_etal11" role="doc-biblioref">2011</a>)</span>. These complex oceanogrpahic processes support a highly productive and diverse assemblage of plankton and fishes, form a rich prey base for at least 25 cetacean species that inhabit the study area <span class="citation" data-cites="carretta_etal23">(<a href="#ref-carretta_etal23" role="doc-biblioref">Carretta et al. 2023</a>)</span>.</p>
</section>
<section id="sample-processing" class="level3">
<h3 class="anchored" data-anchor-id="sample-processing">Sample Processing</h3>
<p>We extracted DNA from all samples using a phenol-chloroform extraction protocol as described in XXX (Shaffer? Valdivia? another?). We then amplified each sample using three different primers: MiFish Universal (hereafter MFU, REF), targeting the 12S region of fishes, which often captures marine mammals; Marine Vertebrates 1 (hereafter MarVer1, REF), targeting the 12S region of marine vertebrates broadly; and D-loop (hereafter DL, REF) targeting the control region of cetaceans specifically. We amplified one technical replicate for MFU, and three technical replicates each for DL and MV1. Our PCR amplification protocols … <strong>MORE HERE FROM MEGAN?</strong></p>
</section>
<section id="bioinformatic-processing" class="level3">
<h3 class="anchored" data-anchor-id="bioinformatic-processing">Bioinformatic Processing</h3>
<p>All raw sequence data were processed using a custom dada2-based pipeline and custom reference database based on NCBI sequence data. The pipeline and code for reference database generation can be found at [GITHUB REPO HERE]. Briefly, we … <strong>MORE HERE FROM PEDRO?</strong></p>
<p>Once the raw sequences were trimmed, filtered, merged, clustered to ASVs, and once chimeras were removed, we assigned ASV taxonomy using … <strong>MORE HERE FROM PEDRO?</strong></p>
<p>For each primer region, we built a custom reference database using all available sequence data from NCBI with the following filtering parameters. First, because the 12S mtDNA region has generally low resolution across many cetacean species, we filtered the reference database to exclude any species not observed in the North Pacific Ocean, excepting a few species known to occasionally visit from nearby regions, e.g.&nbsp;bowhead and beluga whales. We also exclude any of NCBI’s sequence IDs that were not manually validated by a human, due to the fact that NCBI’s automated ID pipeline is still in beta mode and can produce erroneous identifications (REF here?).</p>
</section>
<section id="data-qaqc" class="level3">
<h3 class="anchored" data-anchor-id="data-qaqc">Data QAQC</h3>
<p>We defined a sample as a station/depth/technical replicate combination, and first removed any samples considered to be sequencing failures. Sequencing failures were defined for MV1 and MFU as samples for which no fish were successfully sequenced. Because DL targets cetaceans and excludes fish, we did not define a sequencing failure for this primer and instead included all samples run. Positive detections were defined as samples containing at least one cetacean DNA sequence for each of the three primers, following XXX (REF HERE). We filtered out any non-target (non-cetacean) species, and also removed any detections of bottlenose dolphins and unidentified Balaenopterids or Delphinids (because … ).</p>
<p>To obtain non-detections, we included all MFU and MV1 samples that had at least one marine species detected, but no cetaceans - excluding any samples that completely failed sequencing. Because our DL primer specifically targets a subset of very rare species, we assume that sequencing failure indicates a non-detection - therefore, for this primer we included any station/depth/species combination that was not detected as a non-detect.</p>
</section>
<section id="data-analysis" class="level3">
<h3 class="anchored" data-anchor-id="data-analysis">Data Analysis</h3>
<section id="question-1-does-pod-vary-with-depth" class="level4">
<h4 class="anchored" data-anchor-id="question-1-does-pod-vary-with-depth">Question 1: Does POD vary with depth?</h4>
<p>We addressed this question using generalized additive models (GAMs; <span class="citation" data-cites="Wood2017">Wood (<a href="#ref-Wood2017" role="doc-biblioref">2017</a>)</span>) implemented using the mgcv package in R <span class="citation" data-cites="RCoreTeam2016">(<a href="#ref-RCoreTeam2016" role="doc-biblioref">R Core Team 2016</a>)</span>. GAMs have previously been demonstrated to be an appropriate tool for modeling spatial use in cetaceans (e.g. <span class="citation" data-cites="Becker2016">Elizabeth A. Becker et al. (<a href="#ref-Becker2016" role="doc-biblioref">2016</a>)</span>; <span class="citation" data-cites="becker2020">(<a href="#ref-becker2020" role="doc-biblioref"><strong>becker2020?</strong></a>)</span>; <span class="citation" data-cites="becker2022">(<a href="#ref-becker2022" role="doc-biblioref"><strong>becker2022?</strong></a>)</span>; <span class="citation" data-cites="Becker2019">Elizabeth A. Becker et al. (<a href="#ref-Becker2019" role="doc-biblioref">2019</a>)</span>; <span class="citation" data-cites="abrahmsetal_19c">(<a href="#ref-abrahmsetal_19c" role="doc-biblioref"><strong>abrahmsetal_19c?</strong></a>)</span>). We first collapsed our detection data across all technical replicates at each sampling station and depth and defined a positive detection at that location and depth as detection in at least one technical replicate. We fit all GAMs using 0/1 detections as the response variable with a binomial distribution and logit link function, and estimated model parameters using REML. Models examining depth effect alone were built iteratively, starting with (1) assessing probability of detection (POD) across all species at sampling depths between 1 and 500m, then (2) allowing y-intercept to vary by species, and finally (3) allowing model smooth to vary by species.</p>
<p>In addition to examining depth variability across species-level strata, we examined several other ecological stratification schemes in order to determine whether depth variability in detection probability is driven by (1) evolutionary inertia (i.e.&nbsp;taxonomic family) or (2) preferred prey type.</p>
<p>We assessed model fit using AIC, and model performance using standard metrics including the percent of deviance explained, the area under the receiver operating characteristic curve (AUC; Fawcett 2006 REF), the true skill statistic (TSS, Allouche et al.&nbsp;2006 REF), and a visual comparison of predicted vs.&nbsp;observed detection rates across sampling depths. We also visually compared our model-based predictions of probability of detection across depth with estimates of species-specific time-at-depth as reported in <span class="citation" data-cites="watwood_buonantony12">(<a href="#ref-watwood_buonantony12" role="doc-biblioref">Watwood and Buonantony 2012</a>)</span> and <span class="citation" data-cites="u_s_departmentofthenavy17">(<a href="#ref-u_s_departmentofthenavy17" role="doc-biblioref">Navy 2017</a>)</span>.</p>
</section>
<section id="question-2-does-depth-variability-in-pod-covary-with-observation-bias" class="level4">
<h4 class="anchored" data-anchor-id="question-2-does-depth-variability-in-pod-covary-with-observation-bias">Question 2: Does depth variability in POD covary with observation bias?</h4>
<p>To address this question, we extracted smooths from the best fit model in Q1 and input them into a Bayesian hierarchical occupancy model, developed based on Patin et al.&nbsp;(<em>in prep</em>) to explicitly examine depth variability alongside potential sources of observational bias, including technical replication, primer specificity, sampling volume, and number of freeze/thaw cycles. <strong>MORE HERE</strong></p>
</section>
<section id="question-3-does-depth-variability-in-pod-covary-with-xy-variability" class="level4">
<h4 class="anchored" data-anchor-id="question-3-does-depth-variability-in-pod-covary-with-xy-variability">Question 3: Does depth variability in POD covary with xy variability?</h4>
<p>We returned to the use of GAMs to address Question 3, and again built iterative models in order to examine covariability between depth, latitude, and longitude on a per-species basis. We began by examining whether xy variability explained a significant amount of remaining variability left in the model after depth variability was account for (i.e.&nbsp;POD ~ z + xy), then tested whether x, y, and z covary (i.e.&nbsp;POD ~ xyz). Finally, we allowed the model of xyz covariability to vary idependently by species.</p>
<p>We assessed model fit using AIC, and model performance using percent deviance explained, AUC, and TSS, as in Question 1. We then conducted a visual comparison of the predicted depth of maximum POD across and our observed detection depth within the study area for the most frequently observed baleen whale, dolphin, and beaked whale.</p>
</section>
</section>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<p>We selected 573 water samples collected at 182 unique locations during the 2019 NOAA hake abundance survey (<a href="#fig-bubble" class="quarto-xref">Figure&nbsp;1</a>), targeting 0, 50, 150, 300, and 500 m sampling depths at each station. Where the bottom was less than 500 m deep, we did not collect a sample at 500 m. We amplified and sequenced these samples in 1 technical replicate using MFU, 3 technical replicates using MV1, and 3-5 technical replicates using DL, resulting in a final total of 1559 amplified and sequenced samples.</p>
<div class="cell">
<div id="tbl-sp-detects" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-sp-detects-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Scientific names, common names, abbrevation, taxonomic family, broad prey category, and total number of detections for each cetacean species detected in our study area.
</figcaption>
<div aria-describedby="tbl-sp-detects-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="do-not-create-environment cell caption-top table table-sm table-striped small">
<caption>Table continues below</caption>
<colgroup>
<col style="width: 37%">
<col style="width: 39%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Species</th>
<th style="text-align: center;">Common name</th>
<th style="text-align: center;">Sp. abbrevation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Lagenorhynchus obliquidens</td>
<td style="text-align: center;">Pacific white-sided dolphin</td>
<td style="text-align: center;">Lobl</td>
</tr>
<tr class="even">
<td style="text-align: center;">Megaptera novaeangliae</td>
<td style="text-align: center;">humpback whale</td>
<td style="text-align: center;">Mnov</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Phocoenoides dalli</td>
<td style="text-align: center;">Dall’s porpoise</td>
<td style="text-align: center;">Pdal</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenoptera physalus</td>
<td style="text-align: center;">fin whale</td>
<td style="text-align: center;">Bphy</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Phocoena phocoena</td>
<td style="text-align: center;">harbor porpoise</td>
<td style="text-align: center;">Ppho</td>
</tr>
<tr class="even">
<td style="text-align: center;">Stenella coeruleoalba</td>
<td style="text-align: center;">striped dolphin</td>
<td style="text-align: center;">Scoe</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Grampus griseus</td>
<td style="text-align: center;">Risso’s dolphin</td>
<td style="text-align: center;">Ggris</td>
</tr>
<tr class="even">
<td style="text-align: center;">Orcinus orca</td>
<td style="text-align: center;">killer whale</td>
<td style="text-align: center;">Oorc</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Berardius bairdii</td>
<td style="text-align: center;">northern giant bottlenose whale</td>
<td style="text-align: center;">Bbai</td>
</tr>
<tr class="even">
<td style="text-align: center;">Ziphius cavirostris</td>
<td style="text-align: center;">goose-beaked whale</td>
<td style="text-align: center;">Zcav</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Eschrichtius robustus</td>
<td style="text-align: center;">gray whale</td>
<td style="text-align: center;">Erob</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenoptera musculus</td>
<td style="text-align: center;">blue whale</td>
<td style="text-align: center;">Bmus</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Balaenoptera acutorostrata</td>
<td style="text-align: center;">minke whale</td>
<td style="text-align: center;">Bacu</td>
</tr>
<tr class="even">
<td style="text-align: center;">Lissodelphis borealis</td>
<td style="text-align: center;">northern right whale dolphin</td>
<td style="text-align: center;">Lbor</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Mesoplodon stejnegeri</td>
<td style="text-align: center;">saber-toothed whale</td>
<td style="text-align: center;">Mste</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaena mysticetus</td>
<td style="text-align: center;">bowhead whale</td>
<td style="text-align: center;">Bmys</td>
</tr>
</tbody>
</table>
<table class="do-not-create-environment cell caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 21%">
<col style="width: 22%">
<col style="width: 16%">
<col style="width: 18%">
<col style="width: 6%">
<col style="width: 7%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Family</th>
<th style="text-align: center;">Broad taxon</th>
<th style="text-align: center;">Prey family</th>
<th style="text-align: center;"># Detections</th>
<th style="text-align: center;">DL</th>
<th style="text-align: center;">MFU</th>
<th style="text-align: center;">MV1</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Delphinidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">120</td>
<td style="text-align: center;">19</td>
<td style="text-align: center;">22</td>
<td style="text-align: center;">79</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenopteridae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">94</td>
<td style="text-align: center;">21</td>
<td style="text-align: center;">16</td>
<td style="text-align: center;">57</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Phocoenidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">88</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">5</td>
<td style="text-align: center;">71</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenopteridae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">62</td>
<td style="text-align: center;">5</td>
<td style="text-align: center;">16</td>
<td style="text-align: center;">41</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Phocoenidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">46</td>
<td style="text-align: center;">7</td>
<td style="text-align: center;">6</td>
<td style="text-align: center;">33</td>
</tr>
<tr class="even">
<td style="text-align: center;">Delphinidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">36</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">34</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Delphinidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">33</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">3</td>
<td style="text-align: center;">18</td>
</tr>
<tr class="even">
<td style="text-align: center;">Delphinidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">23</td>
<td style="text-align: center;">6</td>
<td style="text-align: center;">3</td>
<td style="text-align: center;">14</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Ziphiidae</td>
<td style="text-align: center;">Beaked whale</td>
<td style="text-align: center;">squid</td>
<td style="text-align: center;">16</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">3</td>
<td style="text-align: center;">12</td>
</tr>
<tr class="even">
<td style="text-align: center;">Ziphiidae</td>
<td style="text-align: center;">Beaked whale</td>
<td style="text-align: center;">squid</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">8</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Eschrichtiidae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">12</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">8</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenopteridae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">8</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Balaenopteridae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">7</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td style="text-align: center;">Delphinidae</td>
<td style="text-align: center;">Dolphin/Porpoise</td>
<td style="text-align: center;">fish</td>
<td style="text-align: center;">5</td>
<td style="text-align: center;">5</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Ziphiidae</td>
<td style="text-align: center;">Beaked whale</td>
<td style="text-align: center;">squid</td>
<td style="text-align: center;">3</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">3</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balaenidae</td>
<td style="text-align: center;">Baleen whale</td>
<td style="text-align: center;">invert</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">0</td>
</tr>
</tbody>
</table>
</div>
</div>
</figure>
</div>
</div>
<p>Across all samples, we observed 566 positive detections of 16 cetacean species, including 6 baleen whale species, 5 dolphin species, 2 porpoise species, and 3 beaked whale species (<a href="#tbl-sp-detects" class="quarto-xref">Table&nbsp;1</a>). 11 species were observed more than 10 times, and and additional 5 were observed fewer than 10 times. This includes one detection of a bowhead whale (<em>Balaena mysticetus</em>), as well as three detections of Stejneger’s beaked whales (<em>Mesoplodon stejnegeri</em>). <a href="#fig-bubble" class="quarto-xref">Figure&nbsp;1</a> illustrates the distribution of detections across sampling depths, ignoring x-y distribution, for all cetacean species included in the study.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-bubble" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-bubble-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="index_files/figure-html/fig-bubble-1.png" class="img-fluid figure-img" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-bubble-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Left: Distribution of detections across sampling depths for all species included in the study. Species are grouped as follows: Baleen whales includes the Balaenopteridae, Balaenidae, and Eschericthidae; Dolphins/Porpoises includes all Delphinidae and Phocoenidae; Beaked whales includes all Ziphiidae. Right: Geographic distribution of samples used in this study, acquired from water collected during a 2019 survey of hake distribution in the California Current.
</figcaption>
</figure>
</div>
</div>
</div>
<section id="question-1-does-pod-vary-with-depth-1" class="level3">
<h3 class="anchored" data-anchor-id="question-1-does-pod-vary-with-depth-1">Question 1: Does POD vary with depth?</h3>
<p>GAM-based model selection of detection probability across sampling depths negated the null hypothesis of no variation across depth. The best alternative model includes significant smooth terms for depth, species, and depth x species interaction (AIC = 5410). This model explained 14.3% of variance in eDNA detection probability, <em>with an AUC of XXX and TSS of XXXX.</em></p>
<p>Broadly, delphinids were more likely to be detected at the surface, beaked whales at depths &gt; 200m, and baleen whales either at the surface or at depths &gt; 400m (<a href="#fig-H1POD-species" class="quarto-xref">Figure&nbsp;2</a>). While detection data from the five species detected fewer than 10 times were included in our model input, we did not make predictions for any of these species.</p>
<p>Species-specific detections were qualitatively highest at depths where they are predicted to spend the most time among the delphinids. Conversely, most beaked whale and baleen whale species in the study were most likely to be detected deeper than where they spend most of their time (<a href="#fig-H1POD-species" class="quarto-xref">Figure&nbsp;2</a>). Most notably, predicted POD-at-depth plots for two of the three baleen whales indicated a high probability of detection at 500 m, while these two species spend most of their time above ~200 m.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-H1POD-species" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-H1POD-species-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="index_files/figure-html/fig-H1POD-species-1.png" class="img-fluid figure-img" width="960">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-H1POD-species-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Species-specific probability of detection across all depths sampled in this study. Solid line shows POD; shaded region shows 95% confidence interval. Dashed line shows proportional time at depth. Species abbreviation codes can be found in Table 1; plots are colored by taxonomic category as outlined in Table 1.
</figcaption>
</figure>
</div>
</div>
</div>
<p>Models stratifying detections by taxonomic Family and preferred prey type both had lower AIC values than the model stratified by species (AIC = 5810 and 5949, respectively); however, these models still had significant terms for Family and depth x Family interactions (taxonomic model) and for depth, prey type, and depth x prey type interactions (prey type model). <em>Model diagnostics here.</em> Predicted POD curves for each model illustrate varying trends in POD with depth across strata (<a href="#fig-H1POD-family-prey" class="quarto-xref">Figure&nbsp;3</a>). Similar to the model stratified by species, we did not make predictions for Families with fewer than 10 detections using the taxonomic model, which excluded the Balaenidae (bowhead whale family).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-H1POD-family-prey" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-H1POD-family-prey-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="index_files/figure-html/fig-H1POD-family-prey-1.png" class="img-fluid figure-img" width="960">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-H1POD-family-prey-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Left: POD by depth for species stratified by taxonomic Family. Right: POD by depth for species stratified based on broad prey categories.
</figcaption>
</figure>
</div>
</div>
</div>
<section id="question-2-does-depth-variability-in-pod-covary-with-observation-bias-1" class="level4">
<h4 class="anchored" data-anchor-id="question-2-does-depth-variability-in-pod-covary-with-observation-bias-1">Question 2: Does depth variability in POD covary with observation bias?</h4>
<p>Technical replication increases probability of detection.</p>
</section>
<section id="question-3-does-depth-variability-in-pod-covary-with-xy-variability-1" class="level4">
<h4 class="anchored" data-anchor-id="question-3-does-depth-variability-in-pod-covary-with-xy-variability-1">Question 3: Does depth variability in POD covary with xy variability?</h4>
<p>Our best fit model included terms for species, lat/lon, and depth, as well as species x lat/lon, species x depth, and species x depth x lat/lon (AIC = 4926.9976483). Although terms for depth and lat/lon were not significant, removing these terms increased model AIC to 4947. <em>Model AUC and TSS here</em></p>
<p>Predicted 3D probability of detection for the most frequently detected delphinid, baleen whale, and beaked whale varied with latitute and longitude, while adhering to the broad trends identified in Q1 (<a href="#fig-H3PODxy" class="quarto-xref">Figure&nbsp;4</a>). The Pacific white-side dolphin is most likely to be detected near the surface, with max POD varying between depths of 0 m and 500 m. Generally, detections of Pacific white-sided dolphins that were farther south and east were also more likely to be deeper.</p>
<p>Depths of highest POD for humpback whales had two peaks around 0 m and 500 m; and depths of highest POD for Baird’s beaked whales varied between 120 m and 500 m. Lat/lon variability in Baird’s beaked whale detections followed a similar pattern to Pacific white-sided dolphins, with greater likelihood of deeper detections in stations that are more to the south and east. The two peaks in depth of greatest POD for humpback whales were roughly distributed with higher probability of detection around 0 m on the continental shelf, and higher probability of detection around 500 m on the shelf break.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-H3PODxy" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-H3PODxy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="index_files/figure-html/fig-H3PODxy-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-H3PODxy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Depth of maximum probability of detection (POD; top) and width of 95% confidence itervals at max POD depth (bottom) for three species: Bbai = Baird’s beaked whale (<em>B. bairdii</em>), Lobl = Pacific white-sided dolphin (<em>L. obliquidens</em>), Mnov = humpback whale (<em>M. novaegliae</em>).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
</section>
<section id="discussion" class="level2">
<h2 class="anchored" data-anchor-id="discussion">Discussion</h2>
<section id="question-1" class="level4">
<h4 class="anchored" data-anchor-id="question-1">Question 1:</h4>
<ol type="1">
<li><p>Sampling depth matters. To target rare species, optimizing sampling depth could increase detection rates and improve xy distributional models while minimizing cost.</p></li>
<li><p>Sampling depth varies by species, and roughly according to some basic ecological principles, e.g.&nbsp;generally eDNA was found where the animals spend most of their time, and prey type was indicative of POD across depth.</p></li>
<li><p>Notable caveats are: whalefall and poop (the latter mostly noticeable for baleen whales). <em>Include some figures of potential whalefall here or in supplement? Or add this as an explicit method/result? Include poop sinking equations here or in a supplement?</em></p></li>
<li><p>Beaked whale distributions are poorly understood and could be improved the most by eDNA sampling/monitoring. Talk here about how eDNA depth distributions compare to what we know about beaked whale distributions, e.g.&nbsp;two of the detected species are found most often around 300m, where they are thought to take resting dives. (Is this where they spend most of their time according to Navy reports?) Stejnegers was found deeper. Sampling deeper than 500m may increase detection for deeper diving beaked whales, e.g.&nbsp;some of the ones that are almost never detected at the surface. <em>EKJ note: do we think this is because beaked whales are rare rare targets, so their eDNA has a better chance of being detected at depth, where there are fewer competing bits of other cetacean eDNA?)</em></p></li>
</ol>
</section>
<section id="question-2" class="level4">
<h4 class="anchored" data-anchor-id="question-2">Question 2:</h4>
<ol start="5" type="1">
<li>Increasing technical replication (and primer specificity?) increases POD. <em>and this covaries with depth effect?</em></li>
</ol>
</section>
<section id="question-3" class="level4">
<h4 class="anchored" data-anchor-id="question-3">Question 3:</h4>
<ol start="6" type="1">
<li><p>Effect of depth varies across x and y dimensions. EKJ note: if we find this, it would be indicative of oceanographic stuff (upwelling etc) affecting probability of detection</p></li>
<li><p>Over the large geographic range covered by this study and the large range of mobile rare species, we expect that this at least partially reflects true distribution of the animals.</p></li>
<li><p>It could also reflect, e.g.&nbsp;shorter residence time of eDNA in warmer water to the south, or higher probability of eDNA sinking below 500 meters in deeper offshore water. <em>What else could be causing this?</em></p></li>
</ol>
</section>
<section id="marine-mammal-monitoring-lessons-for-edna-experimental-design" class="level4">
<h4 class="anchored" data-anchor-id="marine-mammal-monitoring-lessons-for-edna-experimental-design">Marine Mammal monitoring: lessons for eDNA experimental design</h4>
<ol type="1">
<li><p>Sample at depth where species spends most of its time. If this is unknown, can use broad prey category or taxonomic group as a guide.</p></li>
<li><p>In laboratory processing, sequence longer fragments first, using as few freeze/thaw cycles as possible.</p></li>
</ol>
</section>
</section>
<section id="data-availability" class="level2">
<h2 class="anchored" data-anchor-id="data-availability">Data availability</h2>
<p>All raw sequence data have been uploaded to GenBank (Accessions: XXXX). Detection data, sample metadata, and all code used to generate this manuscript are available at https://github.com/MMARINeDNA/zDistribution.</p>
</section>
<section id="acknowledgements" class="level2">
<h2 class="anchored" data-anchor-id="acknowledgements">Acknowledgements</h2>
</section>
<section id="references" class="level2">
<h2 class="anchored" data-anchor-id="references">References</h2>
</section>
<section id="supplemental-materials" class="level2">
<h2 class="anchored" data-anchor-id="supplemental-materials">Supplemental Materials</h2>
<p>Table S1. Sample metadata.</p>
<p>Table S2. Q1 model AIC and diagnostics</p>
<p>Figure S1. Q1 and Q3 Null hypothesis plots?</p>

</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-auad_etal11" class="csl-entry" role="listitem">
Auad, Guillermo, Dean Roemmich, and John Gilson. 2011. <span>“The <span>California</span> <span>Current</span> <span>System</span> in Relation to the <span>Northeast</span> <span>Pacific</span> <span>Ocean</span> Circulation.”</span> <em>Progress in Oceanography</em> 91 (4): 576–92. <a href="https://doi.org/10.1016/j.pocean.2011.09.004">https://doi.org/10.1016/j.pocean.2011.09.004</a>.
</div>
<div id="ref-Becker2016" class="csl-entry" role="listitem">
Becker, Elizabeth A., Karin A. Forney, Bruce J. Thayre, Amanda Debich, Gregory S. Campbell, Katherine Whitaker, Annie B. Douglas, Anita Gilles, Ryan Hoopes, and John A. Hildebrand. 2016. <span>“Habitat-<span>Based</span> <span>Density</span> <span>Models</span> for <span>Three</span> <span>Cetacean</span> <span>Species</span> Off <span>Southern</span> <span>California</span> <span>Illustrate</span> <span>Pronounced</span> <span>Seasonal</span> <span>Differences</span>”</span> 4 (May): 1–14. <a href="https://doi.org/10.3389/fmars.2017.00121">https://doi.org/10.3389/fmars.2017.00121</a>.
</div>
<div id="ref-Becker2019" class="csl-entry" role="listitem">
Becker, Elizabeth A, Karin A Forney, Jessica V Redfern, Jay Barlow, Michael G Jacox, Jason J Roberts, and Daniel M Palacios. 2019. <span>“Predicting Cetacean Abundance and Distribution in a Changing Climate.”</span> <em>Diversity and Distributions</em> 25 (4): 626–43. <a href="https://doi.org/10.1111/ddi.12867">https://doi.org/10.1111/ddi.12867</a>.
</div>
<div id="ref-carretta_etal23" class="csl-entry" role="listitem">
Carretta, James V., Erin M. Oleson, Karin A. Forney, David W. Weller, Aimee R. Lang, Jason Baker, Anthony J Orr, et al. 2023. <span>“U.<span>S</span>. <span>Pacific</span> Marine Mammal Stock Assessments: 2022.”</span> U.S. Department of Commerce, NOAA Technical Memorandum NMFS-SWFSC-684. <a href="https://repository.library.noaa.gov/view/noaa/51022">https://repository.library.noaa.gov/view/noaa/51022</a>.
</div>
<div id="ref-hickey79" class="csl-entry" role="listitem">
Hickey, Barbara M. 1979. <span>“The <span>California</span> Current System—Hypotheses and Facts.”</span> <em>Progress in Oceanography</em> 8 (4): 191–279. <a href="https://doi.org/10.1016/0079-6611(79)90002-8">https://doi.org/10.1016/0079-6611(79)90002-8</a>.
</div>
<div id="ref-mcclatchie14" class="csl-entry" role="listitem">
McClatchie, Sam. 2014. <span>“Oceanography of the <span>Southern</span> <span>California</span> <span>Current</span> <span>System</span> <span>Relevant</span> to <span>Fisheries</span>.”</span> In <em>Regional <span>Fisheries</span> <span>Oceanography</span> of the <span>California</span> <span>Current</span> <span>System</span>: <span>The</span> <span>CalCOFI</span> Program</em>, edited by Sam McClatchie, 13–60. Dordrecht: Springer Netherlands. <a href="https://doi.org/10.1007/978-94-007-7223-6_2">https://doi.org/10.1007/978-94-007-7223-6_2</a>.
</div>
<div id="ref-u_s_departmentofthenavy17" class="csl-entry" role="listitem">
Navy, U. S. Department of the. 2017. <span>“Dive&nbsp;Distribution&nbsp;and&nbsp;Group&nbsp;Size&nbsp;Parameters&nbsp;for&nbsp;Marine&nbsp;Species&nbsp;Occurring&nbsp;in&nbsp;the&nbsp;u.s.&nbsp;Navy’s&nbsp;Atlantic&nbsp;and&nbsp;Hawaiisouthern&nbsp;California&nbsp;Training&nbsp;and&nbsp;Testing&nbsp;Study&nbsp;Areas.”</span> Newport, Rhode Island: Naval Undersea Warfare Center Division.
</div>
<div id="ref-RCoreTeam2016" class="csl-entry" role="listitem">
R Core Team. 2016. <span>“R: A Language and Environment for Statistical Computing.”</span> R Foundation for Statistical Computing. <a href="https://www.r-project.org/">https://www.r-project.org/</a>.
</div>
<div id="ref-watwood_buonantony12" class="csl-entry" role="listitem">
Watwood, Stephanie L, and Danielle M Buonantony. 2012. <span>“Dive <span>Distribution</span> and <span>Group</span> <span>Size</span> <span>Parameters</span> for <span>Marine</span> <span>Species</span> <span>Occurring</span> in <span>Navy</span> <span>Training</span> and <span>Testing</span> <span>Areas</span> in the <span>North</span> <span>Atlantic</span> and <span>North</span> <span>Pacific</span> <span>Oceans</span>. <span>NUWC</span>-<span>NPT</span> <span>Technical</span> <span>Document</span> 12,085.”</span> Newport, Rhode Island: Naval Undersea Warfare Center Division.
</div>
<div id="ref-Wood2017" class="csl-entry" role="listitem">
Wood, S N. 2017. <em>Generalized <span>Additive</span> <span>Models</span>: <span>An</span> <span>Introduction</span> with <span>R</span></em>. Chapman; Hall/CRC.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>